packages/now-python/src/install.ts

```diff
@@ -0,0 +1,120 @@
+import execa from 'execa';
+import { debug, Meta } from '@now/build-utils';
+const pipPath = 'pip3';
+
+async function isInstalled(dependency: string, cwd: string) {
+  try {
+    await execa('python3', ['-c', `"import ${dependency}"`], {
```

Comment:
@remorses This is returning a false positive.

For example, `python3 -c "import werkzeug"` works every time.

Reply:
Fixed in #3695